gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -128,13 +130,19 @@ def as_api_model(self) -> models.AfmExecution:
         return models.AfmExecution(execution=execution, result_spec=result_spec)
 
 
-ResultSize = Tuple[Optional[int], ...]
+ResultSizeDimensions = Tuple[Optional[int], ...]
+ResultSizeBytes = int
```

Comment:
Is there any benefit from having this alias in place? It holds the same information as variable name so it seems to be redundant. I also do not expect data type will be changed in the future for `ResultSizeBytes`

Reply:
fixed